public_html/wp-content/plugins/wordcamp-payments/includes/reimbursement-request.php

```diff
@@ -289,8 +291,13 @@ function render_status_metabox( $post ) {
 	if ( current_user_can( 'manage_network' ) ) {
 		$current_user_can_edit_request = true;
 	} elseif ( in_array( $post->post_status, $editable_statuses ) ) {
-		$submit_text = esc_html__( 'Submit for Review', 'wordcamporg' );
-		$submit_note = esc_html__( 'Once submitted for review, this request can not be edited.', 'wordcamporg' );
+		if ( can_submit_request( $post ) ) {
+			$submit_text = esc_html__( 'Submit for Review', 'wordcamporg' );
+			$submit_note = esc_html__( 'Once submitted for review, this request can not be edited.', 'wordcamporg' );
+		} else {
+			$submit_note = esc_html__( 'Documentation must be attached and saved to the draft before this request can be submitted.', 'wordcamporg' );
+		}
+
```

Comment:
Do we want to make this DRY also, since it seems identical?

Reply:
`wordcamp-payments/wordcamp-budgets.php` has some shared stuff already, so that might be good.